test/fixtures/mock-cert.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Saw this before I saw the filename and jumped a bit 😅 (Hope it's really a fake one :-p)